AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -74,6 +75,8 @@ @implementation MSDataStore
 - (instancetype)init {
   if ((self = [super init])) {
     _tokenExchangeUrl = (NSURL *)[NSURL URLWithString:kMSDefaultApiUrl];
+
+    // FIXME: move that to document store (task #60212)
```

Comment:
You can remove this FIXME as task https://msmobilecenter.visualstudio.com/Mobile-Center/_workitems/edit/60212 is closed.